integrations/apache/src/apache.go

```diff
@@ -1,19 +1,25 @@
 package main
 
 import (
+	"time"
+
 	sdk_args "github.com/newrelic/infra-integrations-sdk/args"
 	"github.com/newrelic/infra-integrations-sdk/log"
 	"github.com/newrelic/infra-integrations-sdk/sdk"
 )
 
 type argumentList struct {
 	sdk_args.DefaultArgumentList
-	StatusURL string `default:"http://127.0.0.1/server-status?auto" help:"Apache status-server URL."`
+	StatusURL    string `default:"http://127.0.0.1/server-status?auto" help:"Apache status-server URL."`
+	CABundleFile string `default:"" help:"Alternative Certificate Authority bundle file"`
+	CABundleDir  string `default:"" help:"Alternative Certificate Authority bundle directory"`
 }
 
 const (
 	integrationName    = "com.newrelic.apache"
-	integrationVersion = "1.0.0"
+	integrationVersion = "1.1.0"
+
+	defaultHTTPTimeout = time.Second * 1
```

Comment:
maybe we could include the `timeout` flag too? But then the problem of testing it. Not sure. The final idea is to also have `timeout` as configurable.
/cc @areina

Reply:
I just left it as hardcoded to be consistent with the previous implementation and don't loose the actual focus of the task.